shared/src/libs/enums/time-unit.enum.ts

```diff
@@ -0,0 +1,8 @@
+const TimeUnit = {
+  DAY: 'day',
+  HOUR: 'hour',
+  MINUTE: 'min',
+  SECOND: 'sec',
```

Comment:
I guess it would be better to have them in full form
```suggestion
  MINUTE: 'minute',
  SECOND: 'second',
```

Reply:
I did it because I didn`t see the full form anywhere in the design. Need I change it?

Reply:
Yes, I believe it would be more pretty